frontend/vue/components/constants/helper.ts

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Thanks for this @vabarbosa , what do you think moving this file to somewhere like: `frontend/vue/components/helpers/link.ts` because from `constants/helper.ts` could be a bit confusing due to they are methods not constants themselves. 

Apart from that I tested that it's working with new dependency versions so it seems that this could work 🎉

Reply:
We have a file called `utilities.ts` into the `frontend/ts` I think it would be a good place to move this code.